src/java/com/phenix/pct/WSServerProcess.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2005-2017 Riverside Software
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.phenix.pct;
+
+import java.io.File;
+import java.util.List;
+
+import org.apache.tools.ant.BuildException;
+
+public class WSServerProcess extends PCTRun {
+    private boolean webLogError = true;
+
+    public WSServerProcess() {
+        super(false, false);
+    }
+
+    @Override
+    public void setProcedure(String procedure) {
+        throw new BuildException("Unknown attribute");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Unknown attribute" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1192)